apis/v1alpha2/gateway_types.go

```diff
@@ -756,7 +761,7 @@ const (
 	//
 	// Possible reasons for this condition to be False are:
 	//
-	// * "InvalidCertificateRef"
+	// * "InvalidCertificateRefs"
```

Comment:
I'd personally rather keep this reason singular.

Reply:
Reverted.